lms/data_tasks/report/create_from_scratch/01_functions/01_date_functions.sql

```diff
@@ -36,6 +36,9 @@ AS $$
                 ELSE INTERVAL '6 months'
             END
         ))::DATE
+        WHEN resolution = 'week' THEN
```

Comment:
This is a function that takes a "period" and a date and transforms it accordingly.